react/ModalHeader.tsx

```diff
@@ -57,10 +57,10 @@ export default function ModalHeader(props: Props) {
             onClick={handleClose}
             className={`${handles.closeButton} ma0 bg-transparent pointer bw0 pa3`}
           >
-            <IconClose size={24} type="line" />
+            <IconClose size={32} type="line" />
```

Comment:
I would either create an `iconCloseSize` prop or use the `icon-close` block here, I can see people wanting to change this `size`